Feature/user data display

pasternakewa opened the pull request.

pasternakewa committed:
Centre web app content

pasternakewa committed:
Rewrite App component to functional

pasternakewa committed:
Create component displaying users data and init its content

pasternakewa committed:
Move initial data to state

pasternakewa committed:
Add adding and deleting buttons

pasternakewa committed:
Edit button name

pasternakewa committed:
Change date format and displaying; add picture of users

pasternakewa committed:
Style user editing and deleting buttons

pasternakewa committed:
Add displaying button for adding new user

vdanylov commented on src/App.css:
```diff
@@ -2,23 +2,16 @@
-
-.App-intro {
-  font-size: large;
+.user-button:hover {
```
@pasternakewa new line

vdanylov commented on src/App.js:
```diff
@@ -1,10 +1,19 @@
+const App = () => {
+  return (
+    <div
+      style={{
```
@pasternakewa why not in css file?

vdanylov commented on src/components/UsersList.js:
```diff
@@ -0,0 +1,71 @@
+];
+
+const UsersList = () => {
+  const [state, setState] = useState([...users]);
```
@pasternakewa wrong variables names

vdanylov commented on src/components/UsersList.js:
```diff
@@ -0,0 +1,71 @@
+          flexDirection: "column"
+        }}
+      >
+        {state.map((user) => (
```
@pasternakewa use destructuring if You sure about props

pasternakewa committed:
Make requested changes
Move styling to css file, rename variables names, use destructuring when using state props

pasternakewa committed:
Remove unused file
